### DISCUSSION TITLE
Compatibility with Portainer

### DISCUSSION THREAD

**Discussion by xxLeoxx93** [buanet/ioBroker.docker#342] [Questions & Answers / Fragen & Antworten] [answered]
Hey all,

My first time with IOBroker and I fail to get it running. I followed the steps and I'm able to start the container. However I can get it running properly.

I think it related to the port 9000 which I already use for portainer.

Here's the log with the error I get during "Step 3 of 5":
Checking Database connection... Failed.
[DEBUG] Error message: 
[DEBUG] Error: no UUID found
Please check your configuration and try again.

I tried to change the port to 9002 by using this env:
IOB_STATESDB_PORT=9002
IOB_STATESDB_TYPE =jsonl
IOB_STATESDB_HOST=127.0.0.1

But still no luck. Am I missing something?
Thanks!

**Comment by buanet** [comment, marked as answer]
> I tried to change the port to 9002 by using this env:
IOB_STATESDB_PORT=9002
IOB_STATESDB_TYPE =jsonl
IOB_STATESDB_HOST=127.0.0.1

This will not work as this only configures the connection TO the database, not the database port itself.
I`m not sure if the database ports can/ should be changed. 

FYI: Portainer on port 9000 is deprecated. You better switch to 9443 and ssl (https://docs.portainer.io/start/install-ce/server/docker/linux).

Regards,
André
